src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -423,4 +423,17 @@ public Item withLoanType(JsonObject newLoanTypeRepresentation) {
       this.primaryServicePoint,
       newLoanTypeRepresentation);
   }
+
+  public Item withLastCheckIn(JsonObject lastCheckInRepresentation) {
+    Item item = new Item(
+      itemRepresentation.put(ItemProperties.LASTCHECKIN, lastCheckInRepresentation),
```

Comment:
I think it will be better to use 'write' method here. What do you think about it?

Reply:
Put is returning a value, write is not. Here the return value is used as a param,